java/flight/flight-jdbc-driver/pom.xml

```diff
@@ -147,6 +137,28 @@
             <artifactId>flight-sql</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.calcite.avatica</groupId>
+            <artifactId>avatica</artifactId>
+            <version>1.18.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk15on</artifactId>
+            <version>1.61</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
This must be used elsewhere in the project. Let's make sure we use dependencyManagement appropriately.

Reply:
I'm overriding the version.

Reply:
Why is this necessary? Should we update the version generally instead?

Reply:
It added `ErrorCollector.checkThrows` and `assertThrows`: https://github.com/junit-team/junit4/blob/HEAD/doc/ReleaseNotes4.13.md

Reply:
You're right. JUnit 4 is only defined in java/pom.xml to keep running older tests; which also makes me think about maybe starting using JUnit 5 in the future? Not sure about what the implications where when the project started.
I made a simple assertThrows based on JUnit's, which made building with 4.12 possible again.